docs/api/openapi.yaml

```diff
@@ -3881,16 +3881,13 @@ components:
             low:
               type: number
               nullable: false
-            unkown:
+            unknown:
               type: number
               nullable: false
         all_containers_images_whitelisted:
           type: boolean
           nullable: false
           example: false
-        offical:
-          type: boolean
-          nullable: false
```

Comment:
I can't figure out what this is doing here.

Does it plan to go away on its own, or is it accidental? (The correct spelling is earlier in this file)
The PR that introduced it wasn't particularly enlightening.

Reply:
It's accidental, `offical` can be removed.